kaitai_utils.py

```diff
@@ -2,13 +2,22 @@
 import importlib
```

Comment:
In the ticket for this PR adding a test case is mentioned, could you maybe a screenshot that it's working?

Reply:
We are busy adding an entire branch with a running test. I propose we make that test for the final product, since this is really only an intermediate step.